admin/recording_uploader/deploy.py

```diff
@@ -10,31 +12,64 @@
 CURRENT_DIR = pathlib.Path(__file__).parent
 
 
-def main(region_name: str = "us-east-1", guided: bool = True) -> None:
+def main(region_name: str = "us-east-1", destroy: bool = False) -> None:
     """Deploy the uploader to AWS Lambda.
 
     Args:
         region_name (str): The AWS region to deploy the Lambda function to.
-        guided (bool): Whether to use the guided SAM deployment.
+        destroy (bool): Whether to delete the Lambda function.
     """
-    s3 = boto3.client(
-        "s3",
-        region_name=region_name,
-        endpoint_url=f"https://s3.{region_name}.amazonaws.com",
-    )
-    bucket = "openadapt"
-
-    s3.create_bucket(
-        ACL="private",
-        Bucket=bucket,
-    )
-
-    # deploy the code to AWS Lambda
-    commands = ["sam", "deploy"]
-    if guided:
-        commands.append("--guided")
-    subprocess.run(commands, cwd=CURRENT_DIR, check=True)
-    logger.info("Lambda function deployed successfully.")
+    # check if aws credentials are set
+    if os.getenv("AWS_ACCESS_KEY_ID") is None:
+        raise ValueError("AWS_ACCESS_KEY_ID is not set")
+    if os.getenv("AWS_SECRET_ACCESS_KEY") is None:
```

Comment:
Why not read this from `config`?

Reply:
This script is not supposed to be part of the OpenAdapt app, its an admin script that needs to be run by the owner (you) on a machine that has the relevant aws creds in its environment. From the PR description

> From the project root, run python -m scripts.recording_uploader.deploy (ensure that you have the necessary aws creds configured). Once the command completes, note the api url in the output, and paste that onto the config.py's RECORDING_UPLOAD_URL variable

Because config.py is more closely related to settings of the app, I didn't think it'd be useful to add these there.

Reply:
I think we want to read from config.py.

Reply:
How would you suggest a user override these settings? The default values will be empty in `config.py` and `config.defaults.json`, so the only ways are to manually edit the `config.json` file in the `data` folder, or we expose it in the dashboard settings page? A regular user won't be needing to edit this, and might get confused if its in the dashboard settings